package.json

```diff
@@ -42,10 +42,8 @@
     "cronstrue": "^2.11.0",
     "crossfilter": "^1.3.12",
     "csv-parse": "^5.5.6",
-    "d3": "^3.5.17",
-    "d3-array": "^3.1.1",
-    "d3-scale": "^3.3.0",
-    "d3-shape": "^3.1.0",
```

Comment:
This caused unit test errors in Pie Charts where `d3Arc(slice)` throws a Path error, where error messages points to `d3-shape`. Removing this fixes the issue.